content/GStreamerDRPAI.md

```diff
@@ -31,13 +31,23 @@ The plugin also provides you with the following parameters:
 |-----------------------|---------------------|--------:|----------------------------------------------------------------------|
 | **model**             | String              |     --- | The name of the pre-trained model and the directory prefix.          |
 | **multithread**       | Boolean             |    true | Use a separate thread for object detection.                          |
+| **tracking**          | Boolean             |    true | Track detected objects based on their previous locations.            |
 | **log-detects**       | Boolean             |   false | Print detected objects in standard output.                           |
 | **show-fps**          | Boolean             |   false | Render frame rates of video and DRPAI at the corner of the video.    |
 | **stop-error**        | Boolean             |    true | Stop the gstreamer if kernel modules fail to open.                   |
 | **max-video-rate**    | Float [0.001 - 120] |     120 | Force maximum video frame rate using thread sleeps.                  |
 | **max-drpai-rate**    | Float [0 - 120]     |     120 | Force maximum DRPAI frame rate using thread sleeps.                  |
 | **smooth-video-rate** | Float [1 - 1000]    |       1 | Number of last video frame rates to average for a more smooth value. |
 | **smooth-drpai-rate** | Float [1 - 1000]    |       1 | Number of last DRPAI frame rates to average for a more smooth value. |
+| **smooth-bbox-rate**  | Float [1 - 1000]    |       1 | Number of last bounding-box updates to average. (requires tracking)  |
```

Comment:
Which parameter does this average?

Reply:
It is new. Now that I am keeping previous bounding-boxes for IDs, I can make them move smoothly by averaging them out. It adds some delay when moving, but when stationary, bounding-boxes won’t jump around anymore.

Reply:
oh okay, cool, yeah, how many steps can you add in between two inference steps? I assume that it is a function of video fps vs. DRPAI fps, can you explain a bit more how this is done, just to satisfy my interest?